Topics/Tech_Stacks/NumPy.md

```diff
@@ -0,0 +1,199 @@
+# Learning NumPy
+
+## Table of Contents
+
+### [Introduction](#introduction-1)
+
+### [Set Up](#set-up-1)
+
+### [Array Basics](#array-basics-1)
+
+### [Matrices and Operations](#matrices-and-operations-1)
+
+### [Additional Resources](#additional-resources-1)
+
+## Introduction
+This is a short introduction to NumPy, for people looking for a basic overview of the library before doing further research into how to use it, which can be done through visiting some of the additional resources linked at the bottom of this page. Specifically, all the topics covered on this page as well as many more are discussed in both the official beginners guide and the video tutorial linked resources.
+NumPy is a Python library which provides users with new data structures which represent arrays of one or more dimensions, as well as a variety of methods for different mathematical operations which chain together effectively. It is designed for use in engineering, statistical, mathematical, as well as any other computing and/or scientific contexts where complicated operations are to be conducted on data sets of various shapes efficiently.
+
+## Set Up
+In order to use NumPy, it needs to be installed just like any other Python library. The most basic way this can be done is having Python already installed and installing NumPy through the shell using pip or Anaconda:
+Using pip:
+```bash
+> pip install numpy
+```
+Using Anaconda:
+```bash
+> conda install numpy
+```
+For more detailed instructions on ways to install NumPy, there is a linked additional resource at the bottom of the page labelled 'Installing NumPy'.
+
+Finally, in order to use NumPy resources in your Python code, it is important to import the library. The general convention is to give the package the name 'np' in the import as a shorthand, however it is technically not necessary. Code blocks in this page will be written assuming that the import is done as is below:
+```py
+import numpy as np
+```
+
+## Array Basics
+The most basic object to create in NumPy is the array, which is similar to the default Python list, however it is designed to be more memory efficient, reliable, and it comes with a variety of additional methods. An array can be created with an inputted Python list:
+```py
```

Comment:
You already mention the import statement above but I think adding it to the coding example will make the section more modular. It would allow a reader to go directly to the example and see what they need to write to get it working right away.